src/dmclock/test/CMakeLists.txt

```diff
@@ -31,5 +31,7 @@ endif()
   
 add_dependencies(dmclock-tests dmclock)
 
-add_test(NAME dmclock-tests
-  COMMAND $<TARGET_FILE:dmclock-tests>)
+if(NOT FREEBSD)
```

Comment:
@wjwithagen what failure are you seeing in dmclock-tests? could you post a link to it?